Makefile

```diff
@@ -4,10 +4,12 @@ GOARCH ?= amd64
 COMPOSE_PROJECT_NAME := ${TAG}-$(shell git rev-parse --abbrev-ref HEAD)
 BRANCH := $(shell git rev-parse --abbrev-ref HEAD | sed "s!/!-!g")
 ifeq (${BRANCH},master)
-TAG    	:= $(shell git rev-parse --short HEAD)-go${GO_VERSION}
-LATEST_TAG := latest
+	TRACKED_BRANCH := true
```

Comment:
What is going on here?

Reply:
Adding a variable to allow us to tell if this is a tracked branch (i.e. master, or a release branch). This variable will be the gate for adding latest tags.